# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1683452906 -7200
# Parent  70dc38e68208033ab947d74e3b08d05d19130fbb
Bug 1728778 - Re-establish movemail.

Patch will be published later, please contact us for details.

diff --git a/mailnews/moz.configure b/mailnews/moz.configure
--- a/mailnews/moz.configure
+++ b/mailnews/moz.configure
@@ -19,8 +19,11 @@ def mapi_support(arg):
 set_config("MOZ_MAPI_SUPPORT", mapi_support)
 set_define("MOZ_MAPI_SUPPORT", mapi_support)
 
 # =========================================================
 # = OpenPGP integration
 # =========================================================
 
 include("../third_party/openpgp.configure")
+
+set_config("MOZ_MOVEMAIL", target_is_unix)
+set_define("HAVE_MOVEMAIL", target_is_unix)
